app/Http/Controllers/Auth/LoginController.php

```diff
@@ -56,12 +56,18 @@ public function username()
 
     public function login(Request $request, GeneralSettings $general_settings)
     {
+
+
         $validationRules = [
             $this->username() => 'required|string',
             'password' => 'required|string',
         ];
-        if ($general_settings->recaptcha_enabled) {
-            $validationRules['g-recaptcha-response'] = ['required', 'recaptcha'];
+        if ($general_settings->recaptcha_version) {
```

Comment:
No need for the first if statement.
Would rather like a switch statement for other versions